script/release.py

```diff
@@ -101,6 +105,31 @@ def validate_release_env():
         exit("Please get twine via 'pip install gitchangelog' or 'pip install git+git://github.com/vaab/gitchangelog.git' for Python 3.7")
 
 
+def build():
+    shutil.rmtree(BUILT_APPIUM_DIR_PATH, ignore_errors=True)
+    os.system('{} setup.py install'.format(os.getenv('PYTHON_BIN_PATH')))
+
+
+def compare_file_count():
+    def _get_py_files(root_dir):
+        return_files = []
+        for (dir_path, _dirs, files) in os.walk(root_dir):
+            for file_name in files:
+                _, file_extension = os.path.splitext(file_name)
+                if file_extension in ['.py', '.typed']:
+                    return_files.append('{}/{}'.format(dir_path, file_name))
+        return return_files
+
+    original_files = _get_py_files(APPIUM_DIR_PATH)
+    built_files = _get_py_files(BUILT_APPIUM_DIR_PATH)
```

Comment:
we probably don't have to return file names, paths, but only counts

Reply:
I considered which was better, but wanted to check the list when some packages was missed. The listed error message helped me to see the diff in #542 case. No strong opinion, but I won't write the similar code to get files when error happens for now. (Current count is not so huge like over 100)

Reply:
makes sense. In such case I'd consider returning sets instead and only show the actual difference between these sets (using `difference` helper)

Reply:
Good idea